day2.md

```diff
@@ -0,0 +1,171 @@
+
+Many tourists will come here by 5 a.m. for the [live tuna auctions](https://www.youtube.com/watch?v=uv5DloVT3iU). Knowing sleep is the second most important thing, after food, in our lives, we saved ourselves the troubles of thinking wishfully, setting a 4 am alarm, turning it off as soon as it rings, and going right back to sleep. Instead, after a great night of sleep, we woke up naturally around 9 am and headed straight to the Tsukiji fish market with only one clear goal in mind: food!
+
+<img src="day2/14435132_1115751911807835_676936538936024301_o.jpg" width="500"><br/><br/>
+<img src="day2/IMG_20160906_101226.jpg" width="500"><br/>
+
+Walking around the market and surrounding by foods, especially fresh seafoods, got me very excited. The 5-years-old inside me had been screaming, jumping, and dancing all around the whole time as we were walking through the main street and deciding where we should have our first meal of the day. Many of you should be familiar with the analogy that life sometimes is like walking through a forest knowing that you can only pick one fruit from a tree without knowing what's ahead of you nor the opportunity of returning and picking from a tree you have passed. That's exactly how I felt when we were in the fish market and any other times we had to make a decision related to food in Japan! We could only ate so much each day and there were so many yummies surrounding and calling us. As a computer science major, I couldn't help but tried to collect as many relevant data as possible and run it through my real-time "where-should-we-eat" algorithm<sup>[1](#algorithm)</sup>.
+
+I have to apologize that we didn't take many great pictures in the fish market because it was fairly crowded and all the foods there were screaming for our attentions.
+
+<img src="day2/IMG_20160906_101301.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_102331.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_102433.jpg" width="500"><br/>
+Here comes the first thing I got: my all time favorite tamagoyaki（玉子焼き）！
+
+<img src="day2/IMG_20160906_103227.jpg" width="500"><br/>
+<img src="day2/14409662_1115752155141144_7886916382605174951_o.jpg" width="500"><br/>
+<img src="day2/14444831_1115752548474438_2131506267391325625_o.jpg" width="500"><br/>
+<img src="day2/14435156_1115752615141098_910067847601582150_o.jpg" width="500"><br/>
+Then comes scallop sautéed in butter.
+
+<img src="day2/14409862_1115752071807819_502330450151416461_o.jpg" width="500"><br/>
+There are couple small sanding tables in the shop and we just ate there right after it was cooked!<br/>
```

Comment:
- "**standing** tables" instead of "sanding tables" ;)
- Also good practice to add comma to separate the two independent clauses: "...shop**,** and we just..."